### DISCUSSION TITLE
JS client with `unaccent` extension is throwing an error

### DISCUSSION THREAD

**Discussion by victordelajarte** [supabase/supabase#18031] [Questions] [answered]
I've enabled `unaccent` for my DB, it is working fine in the SQL editor but not with the JS client.
`select unaccent('Hôtel') -- => Hotel`

The syntax is probably wrong but I've tried many variations and it's still throwing errors (with a simple Job table containing a `name` text) :
`await supabase.from("Job").select("name, unaccent(name)")` => throws this error : `"Could not find a relationship between 'Job' and 'unaccent' in the schema cache"`

I'd like to use it with textSearch, like this :
`await supabase.from("Job").select("*").textSearch("unaccent(Job.name)", "unaccent('Hôtel')")` => throws this error : `
"column Job.unaccent does not exist"`

**Comment by silentworks** [comment, marked as answer]
You cannot do that `unaccent` is a database functionality so you cannot be calling it inside of your JS client code. You should either do this inside your database using a generated column or by using a view or using a database function.
